cypress/integration/teacher_add_new_unregistered_student.spec.js

```diff
@@ -0,0 +1,189 @@
+import constants from '../support/constants.js'
+import teacherHomePageElements from '../support/elements/teacher_home_page_elements.js'
+import addClassPageElements from '../support/elements/add_class_page_elements.js'
+import studentRosterPageElements from '../support/elements/student_roster_page_elements.js'
+import flashNoticePageElements from '../support/elements/flash_notice_page_elements.js'
+import manageClassesPageElements from '../support/elements/manage_classes_page_elements.js'
+import userHomePageElements from '../support/elements/user_home_page_elements.js'
+import changePasswordPageElements from '../support/elements/change_password_page_elements.js'
+import studentHomePageElements from '../support/elements/student_home_page_elements.js'
+import registerAddStudentPageElements from '../support/elements/register_add_student_page_elements.js'
+import adminPageElements from '../support/elements/admin_page_elements.js'
+import adminSettingsUsersPageElements from '../support/elements/admin_settings_users_page_elements.js'
+
+const STUDENT_TWO_NAME = constants.STUDENT_TWO_LASTNAME + ", " + constants.STUDENT_TWO_FIRSTNAME;
+const STUDENT_TWO_FULLNAME = constants.STUDENT_TWO_FIRSTNAME + " " + constants.STUDENT_TWO_LASTNAME;
+const STUDENT_TWO_USERNAME = constants.STUDENT_TWO_USERNAME;
+const STUDENT_TWO_PASSWORD = constants.STUDENT_TWO_PASSWORD;
+const STUDENT_THREE_NAME = constants.STUDENT_THREE_LASTNAME + ", " + constants.STUDENT_THREE_FIRSTNAME;
+const STUDENT_THREE_FULLNAME = constants.STUDENT_THREE_FIRSTNAME + " " + constants.STUDENT_THREE_LASTNAME;
+const STUDENT_THREE_USERNAME = constants.STUDENT_THREE_USERNAME;
+const STUDENT_THREE_PASSWORD = constants.STUDENT_THREE_PASSWORD;
```

Comment:
As discussed ,
`import * from '../support/constants.js'`
is not allowed. What's allowed is :
`import * as constants from '../support/constants.js'`

Thus, we will end up needing to prefix them with an identifier again. I've changed the name of the 'constants' identifier to 'c' so that it is now shorter.